redhat-developer/developers.redhat.com
# [MWES-3156] Update Akamai to CCUV3

Akamai are ending support for v2 of their FastPurge API which is what we are currently using. Therefore we need to bump it to use v3. The Akamai Plugin supports v3 out-of-the-box, but I have taken the chance here to update the plugin to the latest version and then ensured that we're using v3 by default

This PR is targeting `stage` branch in the first instance so that I can verify the functionality works as expected in an environment that has Akamai support. Additionally I need to do some updates to our Managed Platform environments to support the latest version of Akamai credentials for v3 of the API.

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-3156

### Verification Process

* Build should go green
* View `admin/reports/updates` in Drupal and ensure Akamai plugin is at version `8.x-3.0-alpha4`
* View `admin/config/akamai/config` in Drupal and ensure that `CCU Version` is `Akamai Client CCUv3`
